src/test/regress/expected/fkeys_between_local_ref.out

```diff
@@ -385,6 +385,28 @@ BEGIN;
  reference_table_1 | n          | t
 (8 rows)
 
+ROLLBACK;
+BEGIN;
+  -- disable foreign keys to reference tables
+  SET LOCAL citus.enable_local_reference_table_foreign_keys TO false;
+  CREATE TABLE local_table_6 (col_1 INT PRIMARY KEY);
+  -- first convert local tables to citus local tables in graph
```

Comment:
we can remove comment as it doesn't address

Reply:
copy & paste problem :) fixed now